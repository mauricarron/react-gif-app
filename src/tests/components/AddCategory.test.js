import { shallow } from "enzyme";
import AddCategory from "../../components/AddCategory";

describe("tests on <AddCategory />", () => {
  const setCategories = () => {};

  const wrapper = shallow(<AddCategory setCategories={setCategories} />);

  test("should match Snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });

  test("should handle the input change", () => {
    const input = wrapper.find("input");
    const value = "Input Changed";

    input.simulate("change", { target: { value } });
    expect(wrapper.find("p").text().trim()).toBe(value);
  });
});
