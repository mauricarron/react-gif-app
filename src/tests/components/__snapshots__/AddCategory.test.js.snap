// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on <AddCategory /> should match Snapshot 1`] = `
<form
  onSubmit={[Function]}
>
  <p />
   
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
